Source/WebCore/css/CSSStyleSheet.h

```diff
@@ -60,7 +60,7 @@ class CSSStyleSheet final : public StyleSheet {
     Node* ownerNode() const final { return m_ownerNode; }
     MediaList* media() const final;
     String href() const final;
-    String title() const final { return m_title; }
+    String title() const final { return !m_title.isEmpty() ? m_title : String { }; }
```

Comment:
Nit: I guess I'd write `String()` to construct a null string rather than use the brace form.

Reply:
Don't we have nullString() for null strings? cc @cdumez

Reply:
`nullString()` exists but these two are interchangeable.